src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -1327,9 +1327,15 @@ public static SqlConnectionStringBuilder CreateConnectionStringBuilder(Connectio
                         {

```

Comment:
@VasuBhog please search the codebase for GetConnectionStringParams. There are other places that we should be updating here, from tests to docs.